definitions/infra-container/summary_metrics.yml

```diff
@@ -0,0 +1,103 @@
+containerState:
+  title: Container state
+  unit: STRING
+  tag:
+    key: container.state
+cpuUsedCoresContainer:
+  query:
+    eventId: entityGuid
+    from: ContainerSample
+    select: average(cpuUsedCores)
+  unit: COUNT
+  hidden: true
+cpuUsedCoresK8s:
+  query:
+    eventId: entityGuid
+    from: K8sContainerSample
+    select: average(cpuUsedCores)
+  title: CPU usage (cores)
+  unit: COUNT
+  hidden: true
+cpuUsedCores:
+  derive: "@cpuUsedCoresContainer || @cpuUsedCoresK8s \n"
```

Comment:
\n ?

Something weird happened in this file. Check that there are several quotes with line breaks.

Reply:
ah, the migration script doing weird things, sorry!